.infrastructure/pv.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-todoapp
+  namespace: mateapp
+spec:
+  storageClassName: standard
+  persistentVolumeReclaimPolicy: Delete
+  accessModes:
+    - ReadWriteMany
+  capacity:
+    storage: 1Gi
+  hostPath:
+    path: /data/
```

Comment:
Add Eof(End-of-file), need to add empty line at the end
![image](https://github.com/mate-academy/devops_todolist_kubernetes_task_8_volumes/assets/87720583/a041620a-04a9-46db-a0c0-644e9de3701f)